Makefile

```diff
@@ -17,7 +17,7 @@ IS_GIT_IGNORED = grep -q $(if $1, $1, $@) .gitignore
 VERSION = master
 APP_NAME = $(shell cat package.json 2>/dev/null | $(call JSON_GET_VALUE,name))
 DONE = echo ✓ $@ done
-CONFIG_VARS = curl -fsL https://ft-next-config-vars.herokuapp.com/$1/$(if $2,$2,$(call APP_NAME)).env -H "Authorization: `heroku config:get APIKEY --app ft-next-config-vars`"
```

Comment:
i've checked all our repos, we don't make use of the ability to override the `APP_NAME`.